package de.bowsi.Catchtest;

public class MyException extends Exception{
	
	public MyException() {
		super("Fehler ist aufgetreten!");
		
	}
}
